app/assets/javascripts/calendars/guider-appointments.es6

```diff
@@ -0,0 +1,34 @@
+/* global Calendar, moment */
+{
+  'use strict';
+
+  class GuiderAppointmentsCalendar extends Calendar {
+    constructor(el, config = {}) {
+      const calendarConfig = $.extend(true, {
+        defaultDate: moment(el.data('default-date')),
```

Comment:
Do we need to do this, as you have similar in base class `Calendar` ?

Can this line go into `Calendar`, so that defaultDate is always an instance of `moment` ?